P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -2269,6 +2085,265 @@ occupation and residence.</p>
+      
+      <p>The <gi>listEvent</gi> element is a member of the <ident
+        type="class">model.listLike</ident> class, and may therefore
+        appear inside <gi>standOff</gi>, or wherever else lists are
+        permitted, in the same way as the <gi>listPerson</gi>,
+        <gi>listPlace</gi>, <gi>listOrg</gi> etc. elements described elsewhere in this
+        chapter.</p>
+      <egXML xmlns="http://www.tei-c.org/ns/Examples">
+        <standOff>
+          <listEvent>
+            <event when="1713" ref="http://eco.canadiana.ca/view/oocihm.9_01832">
+              <label>Treaty of Utrecht</label>
+              <!-- In this case, editors decided to understand treaties or legislation as an `event`, which can be considered 
+                a shortcut for several events that led up to their existence or validity (e.g. the signing of a treaty could also
+                be modeled as an event, with the result on paper as an `object` or `bibl`). -->
+              <desc>France ceded to Great Britain its claims to the <orgName>Hudson's Bay Company</orgName> territories in
+                <placeName>Rupert's Land</placeName>, <placeName>Newfoundland</placeName>, and <placeName>Acadia</placeName> and
+                recognized British suzerainty over <orgName type="tribe">the Iroquois</orgName> but retained its other pre-war
+                North American possessions, including 
+                <placeName key="PEI">Île-Saint-Jean</placeName> 
+                (now <placeName key="PEI">Prince Edward Island</placeName>)...</desc>
+            </event>
+            <event when="1774" key="14-GeoIII-c83">
+              <label>Quebec Act</label>
+              <desc>This act of the British Parliament guaranteed free practice of the Catholic faith and restored use of the
+                French Civil Code for private matters throughout the Province of Quebec, which had been expanded in territory
+                following the <ref>Treaty of Paris</ref>.</desc>
+            </event>
+            <event when="1778" ref="http://avalon.law.yale.edu/18th_century/del1778.asp">
+              <label>Treaty of Fort Pitt</label>
+              <desc>Also known as the <name type="event">Treaty with the Delawares</name>, this was the first written treaty
+                between the newly formed <orgName>United States</orgName> and any Native American people, in this case, the
+                <orgName type="tribe">Lenape</orgName> or Delawares.</desc>
+            </event>
+          </listEvent>
+        </standOff>
+      </egXML>
+      <p>The next example demonstrates the use of the
+        <att>where</att> attribute with <gi>event</gi>, coordinated
+        with location information stored in <gi>listPlace</gi>.</p>
+      <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#NDGEOGste-Falls">
+        <standOff>
+          <!-- ========= events ========= -->
+          <listEvent>
```

Comment:
This is a long list of events with, therefore, long lists of related places and persons. Could/Should that be shortened perhaps?
Edit: I see, this is not a new example but it is already _in_ the Guidelines. But still maybe worth shortening for easier reading?